ui/desktop/package.json

```diff
@@ -139,6 +139,10 @@
   },
   "keywords": [],
   "license": "Apache-2.0",
+  "overrides": {
+    "react": "^19.1.1",
+    "react-dom": "^19.1.1"
+  },
```

Comment:
had to do this, but we can remove when this issues is completely resolved https://github.com/idosal/mcp-ui/issues/90